test/account/AccountExecHooks.t.sol

```diff
@@ -183,7 +181,8 @@ contract AccountExecHooksTest is AccountTestBase {
         vm.expectEmit(true, true, true, true);
         emit PluginUninstalled(address(plugin), true);
 
-        vm.prank(address(entryPoint));
-        account1.uninstallPlugin(address(plugin), bytes(""), bytes(""));
+        vm.startPrank(address(entryPoint));
```

Comment:
Is the change from `prank()` to `startPrank()` about being more explicit?

Reply:
No, it's because the subsequent lines have two external calls: one for the install, and one for `pluginManifest()`. The `vm.prank(...);` call only applies to the next external call, which will be the call to `pluginManifest()` instead of the install call, so it stopped working. Switching to `startPrank(...)` was my lazier way of fixing it, instead of putting the `pluginManifest()` call on a new line.

Reply:
Ohh my bad, hadn't noticed the call to get the manifest, sorry!